packages/training/deploy/runpod/setup.py

```
@@ -0,0 +1,251 @@
+        "interruptible": args.spot,
+        "supportPublicIp": True,
+    })
+    
+    print(f"\n✓ Created pod: {pod['id']}")
+    print(f"  Name: {args.name or f'babylon-{args.gpu}'}")
+    print(f"  GPU: {gpu_id}")
+    print(f"  Profile: {profile}")
+    print(f"  Steps: {steps}")
+    print(f"  Min agents/window: {min_agents}")
+    print(f"  Spot: {args.spot}")
+    print(f"\n  View at: https://console.runpod.io/pods")
```

Ruff (0.14.13)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)